.github/workflows/ci.yml

```diff
@@ -20,22 +20,36 @@ defaults:
 
 jobs:
   # Lint can run in parallel with Build.
-  # We also run `yarn install` with the `--prefer-offline` flag to speed things up.
   lint:
     name: Lint
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository
         uses: actions/checkout@v2
 
+      - name: Setup PNPM
+        uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 6.23.6
+
       - name: Setup Node
         uses: actions/setup-node@v2
         with:
           node-version: 16
-          cache: 'yarn'
+          cache: 'pnpm'
+
+      - name: Cache Node Modules
```

Comment:
Is this needed? `setup-node` should be doing this for us automatically based on their README:
<img width="960" alt="Screen Shot 2022-03-08 at 11 56 51 AM" src="https://user-images.githubusercontent.com/622227/157314949-1bf1a2ff-68e6-4929-afde-b940da81a84b.png">

Reply:
Removed these! Good catch.